app/Http/Controllers/ReviewController.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Movie;
+use App\Models\Review;
+use App\Http\Requests\ReviewRequest;
+
+class ReviewController extends Controller
+{
+    public function index()
+    {
+        $reviews = Review::with(['movie'])
```

Comment:
For this one, it may be best to ensure that you only getting the necessary columns to reduce the amount the processing. For example, let's say you have 40000 reviews, for each of these you will fetch the movie with all the columns, but in here you actually only uses maybe 2 of the movies table columns. 

So maybe in the `with` we can have something like 
```
with('movie:id,title,poster')
```
?